doc/user/layouts/shortcodes/replica-options.html

```diff
@@ -2,5 +2,6 @@
 ------------------------------------|------------|-------------------------------------
 `SIZE`                              | `text`     | The size of the replica. For valid sizes, see [cluster replica sizes](/sql/create-cluster-replica#sizes).
 `AVAILABILITY ZONE`                 | `text`     | If you want the replica to reside in a specific availability zone. You must specify an [AWS availability zone ID] in either `us-east-1` or `eu-west-1`, e.g. `use1-az1`. Note that we expect the zone's ID, rather than its name.
-`INTROSPECTION INTERVAL`         | `interval` | The interval at which to collect introspection data. See [Troubleshooting](/ops/troubleshooting) for details about introspection data. The special value `0` entirely disables the gathering of introspection data. Defaults to `1s`.
+`INTROSPECTION INTERVAL`            | `interval` | The interval at which to collect introspection data. See [Troubleshooting](/ops/troubleshooting) for details about introspection data. The special value `0` entirely disables the gathering of introspection data. Defaults to `1s`.
```

Comment:
```suggestion
`INTROSPECTION INTERVAL`            | `interval` | Default: `1s`. The interval at which to collect introspection data. See [Troubleshooting](/ops/troubleshooting) for details about introspection data. The special value `0` entirely disables the gathering of introspection data.
```

Reply:
I made this change (also for `INTROSPECTION DEBUGGING`), but note that there are still a bunch of places in the docs where defaults are specified the old way:

```
❯ rg "Defaults to"
content/sql/show-sources.md
20:_schema&lowbar;name_ | The schema to show sources from. Defaults to `public` in the current database. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).

content/sql/show-sinks.md
23:_schema&lowbar;name_ | The schema to show sinks from. Defaults to `public` in the current database. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).

content/sql/show-objects.md
22:_schema&lowbar;name_ | The schema to show objects from. Defaults to `public` in the current database. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).

content/sql/show-schemas.md
20:_database&lowbar;name_ | The database to show schemas from. Defaults to the current database. For available databases, see [`SHOW DATABASES`](../show-databases).

content/sql/show-indexes.md
20:_schema&lowbar;name_ | The schema to show objects from. Defaults to `public` in the current database if neither on&lowbar;name nor cluster&lowbar;name are specified. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).

content/sql/show-views.md
19:_schema&lowbar;name_ | The schema to show views from. Defaults to `public` in the current database. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).

content/sql/fetch.md
17:_count_ | The number of rows to retrieve. Defaults to `1` if unspecified.

content/sql/create-connection.md
254:`PORT`                                  | `integer`        |          | The port of the AWS PrivateLink service to connect to. Defaults to the broker's port.

content/sql/functions/length.md
19:_encoding&lowbar;name_ | [`string`](../../types/string) | The [encoding](#encoding-details) you want to use for calculating the string's length. _Defaults to UTF-8_.

content/sql/show-tables.md
19:_schema&lowbar;name_ | The schema to show tables from. Defaults to `public` in the current database. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).

content/sql/show-types.md
19:_schema&lowbar;name_ | The schema to show types from. Defaults to `public` in the current database. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).

content/sql/show-materialized-views.md
20:_schema&lowbar;name_ | The schema to show materialized views from. Defaults to `public` in the current database. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).

content/sql/create-source/load-generator.md
33:**TICK INTERVAL**  | The interval at which the next datum should be emitted. Defaults to one second.
34:**SCALE FACTOR**   | The scale factor for the `TPCH` generator. Defaults to `0.01` (~ 10MB).
```

We might want to change those too.

Reply:
Thank you, @teskje! I'm aware, just trying to get ahead and have one less place to patch it when I do. :troll-face-peeking: